plugins/woocommerce-admin/client/wp-admin-scripts/product-tracking/shared.ts

```diff
@@ -406,6 +406,64 @@ const attachAttributesTracks = () => {
  * Attaches product attributes tracks.
  */
 const attachProductAttributesTracks = () => {
+	document
+		.querySelector( '#product_attributes .add_custom_attribute' )
+		?.addEventListener( 'click', () => {
+			recordEvent( 'product_attributes_buttons', {
+				action: 'add_first_attribute',
+			} );
+		} );
+	document
+		.querySelector( '#product_attributes .add_attribute' )
+		?.addEventListener( 'click', () => {
+			// We verify that we are not adding an existing attribute to not
+			// duplicate the recorded event.
+			const selectElement = document.querySelector(
+				'.attribute_taxonomy'
+			) as HTMLSelectElement;
+			// Get the index of the selected option
+			const selectedIndex = selectElement.selectedIndex;
+			if ( selectElement.options[ selectedIndex ]?.value === '' ) {
+				recordEvent( 'product_attributes_buttons', {
+					action: 'add_new',
+				} );
+			}
+		} );
+
+	const attributesSection = '#product_attributes';
+
+	// We attach the events in this way because the buttons are added dynamically.
+	attachEventListenerToParentForChildren( attributesSection, [
+		{
+			eventName: 'click',
+			childQuery: '.woocommerce_attribute_visible_on_product_page',
+			callback: () => {
+				const visibleOnProductPage = document.querySelector(
+					'.woocommerce_attribute_visible_on_product_page'
+				) as HTMLInputElement;
+
+				recordEvent( 'product_attributes_buttons', {
+					action: 'visible_on_product_page',
+					checked: visibleOnProductPage?.checked,
+				} );
+			},
+		},
+		{
+			eventName: 'click',
+			childQuery: '.woocommerce_attribute_used_for_variations',
+			callback: () => {
+				const usedForVariations = document.querySelector(
+					'.woocommerce_attribute_used_for_variations'
+				) as HTMLInputElement;
+
+				recordEvent( 'product_attributes_buttons', {
+					action: 'used_for_variations',
+					checked: usedForVariations?.checked,
+				} );
+			},
+		},
+	] );
```

Comment:
These events were not specified in the PR descripton or the linked issue. And, they do not function properly.

They will always record the value of the *first* matching checkbox on the page. So, if you have more than one attribute, and modify any attribute, it will always record the value of the first attributes checkboxes.

The best solution is probably to modify `attachEventListenerToParentForChildren` so that when it calls the callback, it passes the child element as a parameter, which can then be used to get the correct value.